### ISSUE TITLE
Subprotocol support

### ISSUE THREAD

**Issue by ziad-floosi** [miguelgrinberg/flask-sock#33]
When using this library to provide GraphQL subscriptions support over WebSockets, it seems that it has trouble with handling subprotocols, causing WebSocket connections to fail from clients on Chrome. Are there any plans for subprotocol support in this library?

**Comment by miguelgrinberg** [comment]
I assume you are interested in support for selecting a subprotocol during handshake? That is doable, what I'm probably not going to add here is implementations for actual subprotocols.

**Comment by ziad-floosi** [comment]
Yeah, just selecting a subprotocol. Chrome fails to connect if the handshake reply doesn't contain the subprotocols sent in the `Sec-WebSocket-Protocol` header.

**Comment by miguelgrinberg** [comment]
Okay, I'll look into it.
